src/index.js

```diff
@@ -2,52 +2,40 @@ import styled from 'styled-components'
 import {
   space,
   color,
-  width,
-  fontSize,
-  flexWrap,
-  flexDirection,
-  alignItems,
-  justifyContent,
-  flex,
-  order,
-  alignSelf
+  layout,
+  flexbox,
+  typography
 } from 'styled-system'
+import propTypes from '@styled-system/prop-types'
 
 export const Box = styled('div')({
   boxSizing: 'border-box'
 },
   space,
   color,
-  width,
-  fontSize,
-  flex,
-  order,
-  alignSelf
+  layout,
```

Comment:
I'm not sure why the snapshots are failing, but with Styled System v5, these should probably be grouped with the `compose` utility – see https://styled-system.com/guides/migrating#compose